client/blocks/all-sites/README.md

```diff
@@ -14,7 +14,6 @@ function render() {
 
 ## Props
 
-- `sites (array)` - An array of `site` objects.
```

Comment:
should we also remove the `sites` param in 
```
function render() {
	return <AllSites sites={ sitesArray } />;
}
```

Reply:
Yes. 👍

Reply:
@vykes-mac @vindl I've committed this change here https://github.com/Automattic/wp-calypso/pull/68785/commits/3d40b3213e87bcb8026153568f800ea02a7e960a